src/render/tutorials/configurable-api-gateway/2_0.html.md

```diff
@@ -330,34 +348,34 @@ public final class ProvidersProvider {
     return providers;
   }
 
-  public static JsonArray calculateProviders(JsonObject payload) {
-    return calculateProviders(getResult(payload, "creditCard"), getResult(payload, "paypal"),
-        getResult(payload, "payU"));
+  public static JsonArray processProviders(JsonObject payload) {
+    return processProviders(getResult(payload, "creditCard"), getResult(payload, "paypal"),
+      getResult(payload, "payU"));
   }
 
-
   private static JsonObject getProviderData(JsonObject data, String label, String paymentUrl) {
     return new JsonObject()
-        .put("label", data.getString(label))
-        .put("paymentUrl", data.getString(paymentUrl));
+      .put("label", data.getString(label))
+      .put("paymentUrl", data.getString(paymentUrl));
   }
 
   private static JsonObject getResult(JsonObject payload, String provider) {
     if (payload.containsKey(provider)) {
       return payload.getJsonObject(provider)
-          .getJsonObject("_result");
+        .getJsonObject("_result");
     } else {
       return null;
     }
   }
-
 }
+
 ```
 
 ## External services
 
-We need to define the responses for external services. Our definition calls 4 services. Let's define them.   
-Create `../common-services` directory. Let's put this directory in the parent directory of our current project, since we will be reusing files created there in different tutorials. Now let's create `../common-services/webapi/__files` and put 4 files there:
+We need to define the responses for external services. Our definition calls 4 services. Let's define them.
+
+Create `<KNOTX-STARTERKIT-ROOT>/common-services` directory. Let's put this directory in the main directory of project, since we will be reusing files created there in different tutorials. Now let's create `<KNOTX-STARTERKIT-ROOT>/common-services/webapi/__files` and put 4 files there:
```

Comment:
```suggestion
Create `<KNOTX-STARTERKIT-ROOT>/common-services` directory. Let's put it in the main directory of the project, since we will be reusing files created there in different tutorials. Now create `<KNOTX-STARTERKIT-ROOT>/common-services/webapi/__files` and put 4 files there:
```